package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "quasar-monorepo",
+  "private": true,
+  "packageManager": "yarn@1.22.19",
```

Comment:
I had to specify this because Corepack doesn't let the ecosystem CI script run `yarn` inside a project which specifies `pnpm` as its package manager. Repeating what I mentioned in the PR description, we should consider migrating to pnpm.